packages/notification-service/src/blockscout/transfers.ts

```diff
@@ -147,25 +154,26 @@ export async function handleTransferNotifications(): Promise<void> {
   // To account for this, we save a cache of all blocks already handled in the last |MAX_BLOCKS_TO_WAIT| blocks (|processedBlocks|),
   // so a transaction has that number of blocks to show up on Blockscout before we miss sending the notification for it.
   const blockToQuery = lastBlockNotified - MAX_BLOCKS_TO_WAIT
-  const { goldTokenAddress, stableTokenAddress } = await getTokenAddresses()
-
-  const {
-    transfers: celoTransfers,
-    latestBlock: celoTransfersLatestBlock,
-  } = await getLatestTokenTransfers(goldTokenAddress, blockToQuery, Currencies.GOLD)
-
-  const {
-    transfers: stableTransfers,
-    latestBlock: stableTransfersLatestBlock,
-  } = await getLatestTokenTransfers(stableTokenAddress, blockToQuery, Currencies.DOLLAR)
-
-  const allTransfers = filterAndJoinTransfers(celoTransfers, stableTransfers)
+  const { goldTokenAddress, cUsdTokenAddress, cEurTokenAddress } = await getTokenAddresses()
+
+  const [
+    { transfers: celoTransfers, latestBlock: celoTransfersLatestBlock },
+    { transfers: cUsdTransfers, latestBlock: cUsdTransfersLatestBlock },
+    { transfers: cEurTransfers, latestBlock: cEurTransfersLatestBlock },
+  ] = await Promise.all([
+    await getLatestTokenTransfers(goldTokenAddress, blockToQuery, Currencies.GOLD),
+    await getLatestTokenTransfers(cUsdTokenAddress, blockToQuery, Currencies.DOLLAR),
+    await getLatestTokenTransfers(cEurTokenAddress, blockToQuery, Currencies.EURO),
+  ])
+
+  const allTransfers = filterAndJoinTransfers(celoTransfers, cUsdTransfers, cEurTransfers)
   const newCheckpointBlock = setLastBlockNotified(
-    Math.max(stableTransfersLatestBlock, celoTransfersLatestBlock)
+    Math.max(Math.max(cUsdTransfersLatestBlock, celoTransfersLatestBlock), cEurTransfersLatestBlock)
```

Comment:
nit: `Math.max` works with more than 2 args.
```suggestion
    Math.max(celoTransfersLatestBlock, cUsdTransfersLatestBlock, cEurTransfersLatestBlock)
```